app/controllers/users_controller.rb

```diff
@@ -141,6 +141,16 @@ def update
     end
   end
 
+  def create_release_contract
```

Comment:
@TheDudeWithTheThing would this be the right way to handle this custom action? I'm unsure if i should have the actual method over in the model and call it instead.